static/js/components/CourseList.js

```diff
@@ -44,6 +54,13 @@ class CourseList extends React.Component {
           </li>
         </ul>
         {table}
+      </div>
+      <div className="clear-float">
+        <br/>
+        <p>You need to pass {totalCourses - coursesPassed} more courses
+          before you can apply for <strong>{program.title}</strong> Master's Degree.</p>
+        <Button bsStyle="primary" disabled>Apply for <strong>{program.title}</strong> Master Degree</Button>
+      </div>
       </div>;
```

Comment:
oh I see, I missed the two opening tags on one line.

could you reformat this so it's a little more readable? mainly 1 opening `<div>` tag per line and correct indentation will help a lot.